jvb/src/main/kotlin/org/jitsi/videobridge/xmpp/XmppConnection.kt

```diff
@@ -129,6 +130,17 @@ class XmppConnection : IQListener {
         return true
     }
 
+    /**
+     * Returns ids of [MucClient] that have been added.
+     * @return JSON string of the list of ids
+     */
+    fun getMucClientIds(): String {
```

Comment:
This can be simplified to:
```
    val mucClientIds: String
        get() = JSONArray().apply { addAll(mucClientManager.mucClientIds) }.toJSONString()

```

Reply:
Oh very nice.

Pardon my n00b questions; I've never actually touched Kotlin before.

How do I apply this? Is that a different form of function declaration, or a way to simplify the body? I attempted a few variations but generally ended up with compilation errors.

It did however work if I simplified the whole function to:
```
    fun getMucClientIds(): String {
        return JSONArray().apply { addAll(mucClientManager.mucClientIds) }.toJSONString()
    }
```

Reply:
I've pushed an update with a variation of your simplification that is within my current ability. Happy to make further changes with your guidance.